extension/src/Documentation.ts

```diff
@@ -678,4 +744,21 @@ function codeBlock(code: string): string {
     return result;
 }
 
+function table(innerHtml: string): string {
+    return `${tag('table', innerHtml, true)}\n`;
+}
+function tr(innerHtml: string): string {
+    return `${tag('tr', innerHtml)}\n`;
```

Comment:
```suggestion
    return tag('tr', innerHtml, true);
```

Reply:
that would add a line break after <tr> resulting in 
```
<tr>
<td>asdf</td><tr>

```
instead of 
```
<tr><td>asdf</td><tr>

```

Reply:
Sry, I need new glasses :P